litex/soc/software/include/hw/common.h

```diff
@@ -35,12 +34,22 @@
  * #endif
  */
 
-/* CSR data width (subregister width) in bytes, for direct comparson to sizeof() */
+/* CSR data width (subreg. width) in bytes, for direct comparson to sizeof() */
 #define CSR_DW_BYTES (CONFIG_CSR_DATA_WIDTH/8)
 
 /* CSR subregisters are embedded inside native CPU word aligned locations: */
 #define MMPTR(a) (*((volatile unsigned long *)(a)))
 
+static inline void csr_write_sub(unsigned long v, unsigned long a)
```

Comment:
Maybe `csr_write_part` or `csr_write_section` are better names than `csr_write_sub`?

Reply:
'sub' is for "subregister" (how about 'subreg' then)? I could also go with "slice". Both "part" and "section" are IMHO terms that are too overloaded to ensure we don't confuse future readers of the source code...

Reply:
Or should we call it `csr_write_simple` or to mach the `simple_csrs` used in https://github.com/enjoy-digital/litex/blob/master/litex/soc/interconnect/csr.py? (if simple is confusing we could change it to something better in both software/gateware later)

Reply:
> Or should we call it `csr_write_simple` or to mach the `simple_csrs` ...

That works for me, thanks! Force-pushed an amended commit reflecting that.